build-tools/automation/yaml-templates/commercial-build.yaml

```diff
@@ -49,6 +49,10 @@ steps:
     workingDirectory: ${{ parameters.xaSourcePath }}
     displayName: delete legacy xamarin-android submodule
 
+  - script: rm -rf /Users/builder/android-toolchain/sdk/platforms/android-35
```

Comment:
We should use `$HOME` here to make sure it works on different agent pools, and probably also want to make this non-fatal for when we're building on a fresh image and this path will not exist?

Reply:
Though on second thought, with the change to the `pkgRevision` value of this component in xaprepare I think the version checking logic should handle re-provisioning as needed. Perhaps we don't even need this line anymore?

Reply:
The issue here was that the images had `android-VanillaIceCream Rev 1` installed on them, but it was installed as `android-35 Rev 1`.  Thus `xaprepare` would see it and think the correct thing was already installed.  This caused problems later on.

Lemme try removing this and see if the images have been fixed.

Reply:
Seems to work without this now, removed.  Thanks!